content/en/users/im/cli/_index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "IM CLI"
+type: docs
+weight: 20
+description: >
+  Getting started with IM Command-Line Interface on EGI Cloud
```

Comment:
Is this page just about getting started (an introductory sample)? Or more like everything you need in everyday use to work with the IM CLI (and a link to the full docs)? If the latter, the title is misleading (and occupies too much space). I'd use a title like "The command line interface of Infrastructure Manager".

Reply:
changed.